api/src/main/java/edu/cornell/mannlib/vitro/webapp/controller/individual/IndividualRdfAssembler.java

```diff
@@ -205,16 +207,19 @@ private void removeProhibitedTriples(OntModel o) {
 				String value = stmt.getObject().asLiteral().getString();
 				DataPropertyStatement dps = new DataPropertyStatementImpl(
 						subjectUri, predicateUri, value);
-				RequestedAction pdps = new PublishDataPropertyStatement(o, dps);
-				if (!PolicyHelper.isAuthorizedForActions(vreq, pdps)) {
+				AccessObject pdps = new DataPropertyStatementAccessObject(o, dps);
+				if (!PolicyHelper.isAuthorizedForActions(vreq, pdps, AccessOperation.PUBLISH)) {
 					log.debug("not authorized: " + pdps);
 					stmts.remove();
 				}
 			} else if (stmt.getObject().isURIResource()) {
 				String objectUri = stmt.getObject().asResource().getURI();
-				RequestedAction pops = new PublishObjectPropertyStatement(o,
-						subjectUri, predicateUri, objectUri);
-				if (!PolicyHelper.isAuthorizedForActions(vreq, pops)) {
+				 /** We don't need to know range and domain because publishing never involves faux properties. */
+				Property prop = new Property(predicateUri);
+		        prop.setDomainVClassURI("?SOME_URI");
+		        prop.setRangeVClassURI("?SOME_URI");
```

Comment:
indentation should be aligned with the rest of the file

Reply:
fixed